heliopy/data/ulysses.py

```diff
@@ -229,6 +254,13 @@ def swoops_ions(starttime, endtime, try_download=True):
     extension = '.dat'
     local_base_dir = os.path.join(ulysses_dir, 'swoops', 'ions')
     remote_base_url = ulysses_url
+    units = OrderedDict([('T_p_large', u.K), ('T_p_small', u.K),
+                        ('v_t', u.km / u.s), ('v_r', u.km / u.s),
+                        ('v_n', u.km / u.s), ('r', u.au),
+                        ('n_a', u.cm**-3), ('n_p', u.cm**-3),
+                        ('hlat', u.dimensionless_unscaled),
```

Comment:
The dimensions of this and `'hlon'` below should be degrees (`u.deg`)

Reply:
I'm a bit confused regarding this one so I used a dimensionless quantity! In the README, it states heliospheric latitude and heliospheric longitude. Do you think we may need to add a coordinate system like this https://omniweb.gsfc.nasa.gov/coho/helios/plan_des.html in the future?